service/history/api/startworkflow/api.go

```diff
@@ -506,17 +507,44 @@ func (s *Starter) respondToRetriedRequest(
 	return s.generateResponse(runID, mutableStateInfo.workflowTask, events)
 }
 
+func (s *Starter) getWorkflowStartTime(ctx context.Context, runID string) (start time.Time, retError error) {
+	mutableState, releaseFn, retError := s.getMutableState(ctx, runID)
+	var workflowStartTime time.Time
```

Comment:
I think either use the `start` return value (my preference) or this one - both not have both.

Reply:
+1

Reply:
done in the next rev